src/rollup/RollupEventInbox.sol

```diff
@@ -30,8 +30,12 @@ contract RollupEventInbox is IRollupEventInbox, IDelayedMessageProvider, Delegat
         rollup = address(_bridge.rollup());
     }
 
-    function rollupInitialized(uint256 chainId) external override onlyRollup {
-        bytes memory initMsg = abi.encodePacked(chainId);
+    function rollupInitialized(uint256 chainId, string calldata chainConfig)
+        external
+        override
+        onlyRollup
+    {
+        bytes memory initMsg = abi.encodePacked(chainId, uint8(0), chainConfig);
```

Comment:
Should we `require(chainConfig.length > 0, "EMPTY_CHAIN_CONFIG");` here? I think the nitro node fails to parse the init message otherwise. Of course we can't fully validate it but I think this is a sensible check.

Reply:
what is the uint8(0) here btw?

Reply:
Just a version byte -- if we want to change the init message in the future, it'd be good to have this as an easy way to indicate the version

Reply:
you're right, it's safer to have the config checked, thanks for catching it